jib-maven-plugin/src/test/resources/projects/simple/pom.xml

```diff
@@ -41,6 +41,7 @@
           <to>
             <image>gcr.io/jib-integration-testing/simpleimage:maven</image>
           </to>
+          <args>An argument.</args>
```

Comment:
Didn't know you could actually put it like this.

Reply:
I didn't either, it was an accident but it worked haha.